packages/synapse-react-client/src/lib/style/components/_synapse-nav-drawer.scss

```diff
@@ -51,9 +51,10 @@
     letter-spacing: 0.5px;
   }
   .linkList {
+    margin-top: 15px;
     > a {
       display: block;
-      padding-top: 30px;
+      padding: 15px 0px;
```

Comment:
Fix a small annoyance. Clicking the space just below an item registered a click on the _next_ item. Center the padding on the item instead of just adding top padding to make it more intuitive.

Previous:
![image](https://user-images.githubusercontent.com/17580037/224174220-f694d153-33cb-46ae-8246-58c5f326ac23.png)
Updated:
![image](https://user-images.githubusercontent.com/17580037/224174529-e38a64ea-4385-4b56-866a-ba11bc0e720b.png)